app/ai/voice/agents/breeze_buddy/utils/hold_transfer.py

```
@@ -52,17 +51,15 @@ async def subscribe_and_wait(
-                f"[hold_transfer] Received result on '{channel}': "
+                f"[pubsub] Received result on '{channel}': "
                 f"status={result.get('status')}"
             )
         return result
     except asyncio.TimeoutError:
-        logger.warning(
-            f"[hold_transfer] Timed out after {timeout_seconds}s on '{channel}'"
-        )
+        logger.warning(f"[pubsub] Timed out after {timeout_seconds}s on '{channel}'")
         return None
     except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)